packages/runtime/test/src/parser/parser.ts

```diff
@@ -394,9 +394,12 @@ describe("Parser", function () {
     });
 
     it("should return correct objects for `txn`", () => {
-      const res = opcodeFromSentence(["txn", "Fee"], 1, interpreter);
-      const expected = new Txn(["Fee"], 1, interpreter);
+      let res = opcodeFromSentence(["txn", "Fee"], 1, interpreter);
+      let expected = new Txn(["Fee"], 1, interpreter);
+      assert.deepEqual(res, expected);
 
+      res = opcodeFromSentence(["txn", "Accounts", "1"], 1, interpreter);
+      expected = new Txn(["Accounts", "1"], 1, interpreter);
```

Comment:
why do we need to update these tests ? aren't we testing the array part in opcode-list.ts ?
this tests was only to check if correct object us created.

Reply:
This is related to `assertLen` in constructor. Since we are passing a txField of array type we should have 2 args now, otherwise 1. Hence added two tests - a) where num of args = 1 with `Fee`, and b) where num of args = 2 with `Accounts/ApplicationArgs`. So this is related to creation of object only, not execution.
In opcode-list we test the execution (where we check if correct value was pushed to stack or not)

Reply:
ok

Reply:
please add test for `ApplicationArgs`.

Reply:
ok, it will be similar to Accounts though